projects/canopy/src/lib/table/table.interface.ts

```diff
@@ -8,3 +8,9 @@ export interface TableColumn {
   align: AlignmentOptions;
   showLabel: boolean;
 }
+
+export enum TableFullSizeBreakpoints {
```

Comment:
We have an ongoing discussion around enums vs union types that we haven't really resolved yet. I wouldn't hold this PR up for it but any input you might have into it would be great.

Reply:
I think there are good reasons not to use enums (such as them being compiled), but I do like the syntax. Objectively, with all things considered, we should probably switch to union types IMO. For now though, this at least meets the precedent used in the radio group stack